src/libs/actions/User.js

```diff
@@ -1,8 +1,29 @@
 /* eslint-disable  import/prefer-default-export  */
 
+import _ from 'underscore';
+import lodashGet from 'lodash.get';
 import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
 import * as API from '../API';
+import {signIn} from './Session';
+
+/**
+ * Changes a password for a given account
+ *
+ * @param {String} oldPassword
+ * @param {String} password
+ * @param {String} [twoFactorAuthCode]
+ */
+function changePassword(oldPassword, password, twoFactorAuthCode) {
+    API.ChangePassword({oldPassword, password}).then((response) => {
+        // If we've successfully authenticated the user, ensure sign them in so they don't get booted out
+        if (response.jsonCode === 200) {
+            signIn(password, twoFactorAuthCode);
+        } else {
+            console.error('Could not change password', response);
```

Comment:
NAB, these logs seem unnecessary. Yes to not processing a bad response code, but less sure about logging something. Not really sure what the original thought process behind the first one was.

Reply:
Ah, yeah I was just following what I saw in `User.js` for `getBetas()` - however, if we feel it's unnecessary I'm totally fine with removing them,